src/main/kotlin/org/assertj/generator/gradle/AssertJGeneratorGradlePlugin.kt

```diff
@@ -67,6 +67,9 @@ open class AssertJGeneratorGradlePlugin @Inject internal constructor(
 
     // Create a new task for the source set
     val generationTask = project.tasks.register<AssertJGenerationTask>(generateTaskName, objects, this)
+    generationTask.configure { task ->
+      task.classDirectories.srcDir(this.java.classesDirectory)
+    }
```

Comment:
I'll follow-up here to allow using `Kotlin` sources in a similar fashion. This is something I'd like available for other projects. :shipit: